lib/omniauth/strategies/line.rb

```diff
@@ -5,7 +5,7 @@ module OmniAuth
   module Strategies
     class Line < OmniAuth::Strategies::OAuth2
       option :name, 'line'
-      option :scope, 'profile openid'
+      option :scope, 'profile openid email'
```

Comment:
I think the problem is making it default will make it harder when prototyping. 
I wonder if it could be made an easy option...

Reply:
I have no idea what I was talking about ...